DeepSpeech.py

```diff
@@ -960,6 +1027,54 @@ def main(_):
         tfv1.reset_default_graph()
         do_single_file_inference(FLAGS.one_shot_infer)
 
+    if FLAGS.finetune_lm_csv_files:
+        original_report_count = FLAGS.report_count
+        FLAGS.report_count = 0
```

Comment:
Instead of overwriting the global flag, refactor the called functions to be able to take these values as parameters and pass them in.

Reply:
And since this is just bypassing the report file generation done by `test`, you could just call `evaluate` directly, as that's all `test` does.